### ISSUE TITLE
2.8.6 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3183]
- #2759
- #3105
- #2966
- #3166
- #3154 
- #3170 
- #3111

## Pick list
```
git cherry-pick 43489ec92de904167383e0e7de7ab2b284e34bc6
git cherry-pick 132c5d5f138b26012a2be64b07ca194793866fc0
git cherry-pick 46d129436aec7e9ad06812610e9498540f477377
git cherry-pick d829e5d5d056c144da5532bd822ae087c9031420
git cherry-pick 67d67c08cdad61b838d49696aac033476d79048e
git cherry-pick 3f6ff27dd1040f336ca6215770b25904b0b052e3
git cherry-pick 16496cf90f770990c92a970ff68419584d828e5b
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.8.5..HEAD --pretty='format:- %s'`
```
- Closes #1428 Carousel off center at display size above 1750 px wide (#3170)
- Closes #3153 Duplicate H1 on person pages tripping up some crawlers (#3154)
- Closes #3120: Adding viewsreference patch to fix cache issues for views without empty handling. (#3166)
- Fixes #2964, Fixes #2957 Add Filters on Published/Approved Events (#2966)
- Close #3104 fix typo in az_news_feeds schema. (#3105)
- Closes #2758: Add patch to address Easy Breadcrumb deprecation warnings. (#2759)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.8.6
